code/modules/reagents/reagent_containers/food/condiment.dm

```diff
@@ -580,3 +580,58 @@
 /obj/item/weapon/reagent_containers/food/condiment/chillwax/New()
 	..()
 	reagents.add_reagent(CHILLWAX, 50)
+
+//////////////////////////////////////////////////////////////////////////
+
+/obj/item/weapon/reagent_containers/food/condiment/small
+	possible_transfer_amounts = list(1, 5)
+	amount_per_transfer_from_this = 1
+	var/trash_type
+
+/obj/item/weapon/reagent_containers/food/condiment/small/on_reagent_change()
+	if(reagents.reagent_list.len <= 0)
```

Comment:
You can avoid rightward creep by combining these two if statements; you can also use is_empty() here